app/src/main/java/com/sdp/movemeet/Backend/Serialization/ActivitySerializer.java

```diff
@@ -50,9 +50,14 @@ public static Activity deserialize(Map<String, Object> data) {
                 (String) data.get(ADDRESS_KEY),
                 (Date) data.get(CREATION_KEY)
         );
+
+        String path = (String) data.get(DOCUMENT_PATH_KEY);
+        if (path != null) act.setDocumentPath(path);
```

Comment:
Is it problematic if the path is null ? What would happen then ? And how would that be possible ?

Reply:
I guess it would throw a kind of "StorageException" error. At least with Firebase Storage (which is used to store BLOB files on Firebase and is hence not really the same service as Firebase Firestore), this is the kind of error I get when the user access his profile in "UserProfile" because, initially, he has no profile picture and the path leading to this picture hence doesn't exist yet.

Reply:
Ok I see, but then maybe it would be a good idea to catch this exception ? If we check if the path is not null, I think it would be a good idea to have an "else" case as well